tests/unit/utilities/will-interrupt-process-test.js

```diff
@@ -160,16 +160,43 @@ describe('will interrupt process', function() {
           isTTY: true,
           on: td.function(),
           setRawMode: td.function(),
+          removeListener: td.function(),
         },
       });
 
       let windowsCtrlCTrap = td.matchers.isA(Function);
 
       willInterruptProcess.addHandler(cb);
-      td.verify(process.stdin.setRawMode(true));
       td.verify(process.stdin.on('data', windowsCtrlCTrap));
     });
 
+    it('adds and reverts rawMode on Windows', function() {
+      Object.defineProperty(process, 'platform', {
+        value: 'win',
+      });
+
+      Object.defineProperty(process, 'stdin', {
+        value: {
+          isRaw: false,
+          isTTY: true,
+          on: td.function(),
+          setRawMode: td.function(),
+          removeListener: td.function(),
+        },
+      });
+
+      willInterruptProcess.addHandler(cb);
+      td.verify(process.stdin.setRawMode(true));
+
+      willInterruptProcess.removeHandler(cb);
+      td.verify(process.stdin.setRawMode(false));
+
+      td.verify(process.stdin.setRawMode(), {
+        ignoreExtraArgs: true,
```

Comment:
After quick check of `testdouble` docs I couldn't find a simple way to verify if `setRawMode` was called exactly 2 times, the first one with `true` and second one with `false`. 

I ended up with this approach. Incoming arguments and invocation order is tested above on the lines 189 and 192. The line with `ignoreExtraArgs` ensures that we call `setRawMode` exactly 2 times. 

I'm glad to improve here but not sure how to better express it with `testdouble`.